src/utils/environment.js

```diff
@@ -0,0 +1,22 @@
+const Environment = {
+  getMongoUrl() {
+    const value = process.env.DB_STRING;
+    if (!value) {
+      throw new Error('DB_STRING required')
+    }
+
+    return value
+  },
+
+  getPort() {
+    return process.env.PORT || 3000
+  },
+
+  getNodeEnv() {
+    return process.env.NODE_ENV
+  },
+}
+
+module.exports = {
+  Environment,
+}
```

Comment:
I've put all environment variables here instead of accessing individually. This helps keep the code clean and controls where environment variables are used. It also helps with readability so you know all variables that can be assigned and retrieved.